app/models/event/accessioning_event.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class Event::AccessioningEvent < Event
+  def self.created_accession_number!(eventable, accession_type, user)
+    create!(
+      eventful: eventable,
+      message: "Created #{accession_type} accession number",
+      family: 'accessioning',
+      of_interest_to: 'administrators',
+      created_by: user&.login
+    )
+  end
+
+  def self.updated_accession_number!(eventable, accession_type, user)
+    create!(
+      eventful: eventable,
+      message: "Updated #{accession_type} accession data",
+      family: 'accessioning',
+      of_interest_to: 'administrators',
+      created_by: user&.login
+    )
+  end
```

Comment:
What has been updated here exactly? The accession number itself? Or metadata associated with the accessioned entity? Either the method name or the 'message' could be a little misleading.